docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb

```diff
@@ -1,346 +1,348 @@
-    "\n",
-    "### Reading from CSV Files\n",
-    "Woodwork provides the ability to read data directly from a CSV file into a Woodwork DataFrame. The helper function used for this, `woodwork.read_file`, currently only reads the data into a pandas DataFrame. At some point, this limitation may be removed, allowing data to be read into a Dask or Koalas DataFrame. For now, only pandas DataFrames can be created with this function.\n",
-    "\n",
-    "### Sorting DataFrame on Time Index\n",
-    "When initializing with a time index, Woodwork, by default, will sort the input DataFrame first on the time index and then on the index, if specified. Because sorting a distributed DataFrame is a computationally expensive operation, this sorting is performed only when using a pandas DataFrame. If a sorted DataFrame is needed when using a Dask or Koalas, the user should manually sort the DataFrame as needed.\n",
-    "\n",
-    "### Equality of Woodwork DataFrames\n",
-    "In order to avoid bringing a Dask DataFrame into memory, Woodwoork does not consider the equality of the data when checking whether Woodwork Dataframe initialized from a Dask or Koalas DataFrame is equal to another Woodwork DataFrame. This means that two DataFrames with identical names, columns, indices, semantic tags, and LogicalTypes but different underlying data will be treated as equal if at least one of them uses Dask or Koalas.\n",
-    "\n",
-    "### LatLong Columns\n",
-    "When working with the LatLong logical type, Woodwork converts all LatLong columns to a standard format of a tuple of floats for Dask DataFrames and a list of floats for Koalas DataFrames. In order to do this, the data is read into memory, which may be problematic for large datatsets.\n",
-    "\n",
-    "### Integer Column Names\n",
-    "Woodwork allows column names of any format that is supported by the DataFrame. However, Dask DataFrames do not currently support integer column names.\n",
-    "\n",
-    "### Setting DataFrame Index\n",
-    "When specifying a Woodwork index with a pandas DataFrame, the underlying index of the DataFrame will be updated to match the column specified as the Woodwork index. When specifying a Woodwork index on a Dask or Koalas DataFrame, however, the underlying index will remain unchanged.\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "`df = df.koalas.attach_id_column('distributed-sequence', 'index_col_name')`."
-   ]
-  }
- ],
- "metadata": {
-  "celltoolbar": "Raw Cell Format",
-  "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
-  },
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.8.2"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 4
+    "cells": [
+        {
```

Comment:
Seems like the indentation got changed when you added the entry to the metadata, so the diff is showing the whole file and not just the small change. Was this an automatic formatting thing?

Reply:
It doesn't seem to be impacting the docs build, but probably best to be able to review just the diff of interest.

Reply:
Yep it's an automatic formatting thing

Reply:
Is there a way I can save without formatting just once?